src/components/common/NetworkSelector/__tests__/NetworkMultiSelector.test.tsx

```diff
@@ -32,31 +32,31 @@ describe('NetworkMultiSelector', () => {
       .with({ chainId: '1' })
       .with({ chainName: 'Ethereum' })
       .with({ shortName: 'eth' })
-      .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
+      .with({ features: [FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
       .build(),
     chainBuilder()
       .with({ chainId: '10' })
       .with({ chainName: 'Optimism' })
       .with({ shortName: 'oeth' })
-      .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
+      .with({ features: [FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
       .build(),
     chainBuilder()
       .with({ chainId: '100' })
       .with({ chainName: 'Gnosis Chain' })
       .with({ shortName: 'gno' })
-      .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
+      .with({ features: [FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
       .build(),
     chainBuilder()
       .with({ chainId: '324' })
       .with({ chainName: 'ZkSync Era' })
       .with({ shortName: 'zksync' })
-      .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL] as any })
+      .with({ features: [FEATURES.COUNTERFACTUAL] as any })
       .build(),
     chainBuilder()
       .with({ chainId: '480' })
       .with({ chainName: 'Worldchain' })
       .with({ shortName: 'wc' })
-      .with({ features: [FEATURES.SAFE_141, FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
+      .with({ features: [FEATURES.COUNTERFACTUAL, FEATURES.MULTI_CHAIN_SAFE_CREATION] as any })
```

Comment:
Nit: as multichain is >=1.4.1 specific, I think we should also set the `recommendedMasterCopyVersion` to `'1.4.1'` in these mocks.